tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -485,8 +485,10 @@ class TestBuildCommand_Dotnet_cli_package(BuildIntegBase):
         [
             ("dotnetcore2.1", "Dotnetcore2.1", None),
             ("dotnetcore3.1", "Dotnetcore3.1", None),
+            ("dotnet5.0", "Dotnet5.0", None),
```

Comment:
Not needed. These test are for zip package types.

Reply:
Fixed.